### ISSUE TITLE
i-kyr.gr

### ISSUE THREAD

**Issue by patrickdrd** [kargig/greek-adblockplus-filter#40]
```
||i-kyr.gr/wp-content/uploads/*/banner*$image
||i-kyr.gr/wp-content/uploads/*/alpha*$image
i-kyr.gr##[src*="banner"]
i-kyr.gr##[src*="alpha"]
```

**Comment by kargig** [comment]
should be fixed by 722c176462b9fa326bbccd2e4599642891e1fe69

**Comment by patrickdrd** [comment]
network filters are always preferable
